TechnologyAssessmentApp/TechnologyAssessmentApp/POP Pattern/MainViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  MainViewController.swift
+//  TechnologyAssessmentApp
+//
+//  Created by Kshitij Godara on 24/08/18.
+//  Copyright © 2018 Nagarro. All rights reserved.
+//
+
+import UIKit
+
+class FoodImageView: UIImageView, Shakeable {
```

Comment:
this is great 👍 But I will put this in a different file since it's nothing related to MainViewController. We want this to be reusable.